dotCMS/src/main/java/com/dotmarketing/common/db/DotConnect.java

```diff
@@ -23,6 +23,9 @@
 import com.dotmarketing.util.json.JSONObject;
 import static com.dotcms.util.CollectionsUtils.*;
 
+import com.dotcms.business.CloseDBIfOpened;
+import com.dotcms.business.WrapInTransaction;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotcms.business.WrapInTransaction'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)